### ISSUE TITLE
Object / Sqlvariant type not being added in scripts

### ISSUE THREAD

**Issue by moxplod** [MoonStorm/Dapper.FastCRUD#24]
I have a property 

`public object FIELD_VALUE { get; set; }`

which maps to a DB column FIELD_NAME which is of type sql_variant not null.

That column gets ignored when the insert or update statement is generated. 
Works fine in select statements.

**Comment by MoonStorm** [comment]
I don't believe it's being picked up in selects either. If you look in the ``Configuration/OrmConventions.cs`` file, ``IsSimpleSqlType`` method, it should have failed the check for a valid property. Unless of course you've set the entity manually or you've overridden that method or ``GetEntityProperties``.

**Comment by moxplod** [comment]
You are right. I just realized i am going through a custom sql that i generate and go through Dapper directly instead of going through FastCrud.

What do you recommend i use here to get sql_variant to work properly? I can always override the IsSimpleSqlType to allow objects. That could have other side effects though.

**Comment by MoonStorm** [comment]
I'd override ``GetEntityProperties`` and add it to the enumerable returned by the base class when you encounter your entity type. Or append it to the default properties in case you use fluent registration.

Whatever the case, just get it in there. I'm very curious to hear what happens when a generic object is being used as a property type :wink:

**Comment by moxplod** [comment]
Ended up overriding GetEntityProperties. Added a required attribute to FIELD_VALUE. And forced the required properties to be added regardless if they meet the IsSimpleSqlType criteria or not.

Legacy column i have to support. Will let you know if it works alright, as i think previously L2S was handling converting sql_variant to object. Not sure what will end up happening here.

**Comment by MoonStorm** [comment]
What's the general understanding of that ``Required`` attribute in other ORMs? I saw it in someone else's code. Is it some sort of opt-in approach rather than the opt-out we're using here?

**Comment by moxplod** [comment]
Not sure what ORM's do in general - but in my case i am using it to force it into the queries.
We can also check at the FastCrud level if they are null and throw an exception. Which most ORM's actually do.
